generic/tests/ethtool.py

```diff
@@ -94,8 +99,14 @@ def ethtool_save_params(session):
         return ethtool_get(session)
 
     def ethtool_restore_params(session, status):
-        error_context.context("Restoring ethtool configuration", logging.info)
-        ethtool_set(session, status)
+        cur_stat = ethtool_get(session)
+        logging.debug("Current offload status: %s", str(cur_stat))
+        logging.debug("offload status want to be set: %s", str(status))
```

Comment:
Well these messages are not valuable for product environment since usually we do not take care of that.